legal-api/src/legal_api/resources/v2/business/business_filings/business_filings.py

```diff
@@ -265,6 +265,37 @@ def patch_filings(identifier, filing_id=None):
     return jsonify(filing.json), HTTPStatus.ACCEPTED
 
 
+@bp.route('/filings/search', methods=['POST'])
+@cross_origin(origin='*')
+@jwt.has_one_of_roles([UserRoles.staff])
+def get_fed_filing():
+    """Return a single FED filing."""
+    try:
+        json_input = request.get_json()
+        filing_id = json_input.get('filingId', None)
+
+        if not filing_id:
+            return {'message': 'Expected a filing ID in the request body.'}, HTTPStatus.BAD_REQUEST
+
+        filing_query = db.session.query(Filing).filter(Filing.id == filing_id).first()
+
+        if not filing_query:
+            return {'message': f'Filing with ID {filing_id} not found.'}, HTTPStatus.NOT_FOUND
+
+        filing_data = filing_query.json
+
+        effective_date = datetime.datetime.fromisoformat(filing_data['filing']['header'].get('effectiveDate', None))
+
+        if effective_date > datetime.datetime.utcnow():
+            return jsonify(filing_data), HTTPStatus.OK
```

Comment:
Does this return the filing JSON or the full filing object like this?

![image](https://github.com/user-attachments/assets/d955467f-52e9-4b23-8492-defbcea61dbd)

I ask because we want to make sure the business identifier is part of the response.

Reply:
Or, should we just have the business identifier in the response? depending on use cases

Reply:
![image](https://github.com/user-attachments/assets/89bb3a9a-7dbc-4d6d-8e8e-e4c4f795b92a)

Reply:
** excellent **